client/sites/marketing/tools/index.tsx

```diff
@@ -43,9 +76,21 @@ export default function MarketingTools() {
 			<PageViewTracker path="/marketing/tools/:site" title="Marketing > Tools" />
 
 			<MarketingToolsHeader handleButtonClick={ handleBusinessToolsClick } />
-
+			<Search
+				className={ clsx( 'search-categories__searchbox', {
+					'search-categories__searchbox--mobile': ! isDesktop(),
```

Comment:
the `search-categories__searchbox`  class feels unintuitive. Perhaps `marketing-features__searchbox` would be more accurate?

Reply:
Agreed, changing it.

Reply:
I have renamed the classes to `marketing-tools__searchbox` that aligns with the name of the component, please let me know if you still prefer `marketing-features__searchbox` and I will change it quickly. Change here 7dc8cf6